dev-tools/src/main/resources/ant/integration-tests.xml

```diff
@@ -227,6 +227,27 @@
 
       <extract-pid file="@{es.pidfile}" property="integ.pid"/>
       <echo>Shutting down external node PID ${integ.pid}</echo>
+      <!-- verify with jps that this actually is the correct pid.
+      See if we can find the line "pid org.elasticsearch.bootstrap.Elasticsearch" in the output of jps -l.-->
+      <exec executable="jps">
```

Comment:
I would maybe add failOnError=true. And i wonder if its better to set a property like this first, and use it as the executable:

```<property name=jps_executable location="${JAVA_HOME}/bin/jps"/>```

This could make it less finicky to things like build servers, as well as ensure we are using the matching jps from the jvm we used to launch ES.